examples/fluids/qfunctions/newtonian.h

```diff
@@ -373,65 +212,31 @@ CEED_QFUNCTION(RHSFunction_Newtonian)(void *ctx, CeedInt Q,
 
     // Total flux
     CeedScalar Flux[5][3];
-    for (CeedInt j=0; j<3; j++) {
-      Flux[0][j] = F_inviscid[j].density;
-      for (CeedInt k=0; k<3; k++)
-        Flux[k+1][j] = F_inviscid[j].momentum[k] - stress[k][j];
-      Flux[4][j] = F_inviscid[j].E_total + Fe[j];
-    }
+    FluxTotal(F_inviscid, stress, Fe, Flux);
 
-    for (CeedInt j=0; j<3; j++) {
-      for (CeedInt k=0; k<5; k++) {
+    for (CeedInt j=0; j<3; j++)
+      for (CeedInt k=0; k<5; k++)
         Grad_v[j][k][i] = wdetJ * (dXdx[j][0] * Flux[k][0] +
                                    dXdx[j][1] * Flux[k][1] +
                                    dXdx[j][2] * Flux[k][2]);
-      }
-    }
```

Comment:
Any strong preference for curly braces vs not? I kinda prefer without when possible. There are a few other places in the code where we don't have curly braces on double for-loops, which is mainly why I ask.

Reply:
Not a big deal to me. Rust requires them always.

Reply:
In Ratel I adopted the PETSc convention - {} required for all multi line statements.

So
good:
```C
for (i = 0; i < 42; i++) total += array[i];
```
good:
```C
for (i = 0; i < 42; i++) {
  total += array[i];
}
```
bad:
```C
for (i = 0; i < 42; i++)
  total += array[i];
```

I mostly did that because I have screwed it up a couple of times

Reply:
I was trying to make the style consistent throughout that file but if you are seeing more of those curly braces, then I have missed them.

Reply:
haha, I have been trying to push code I write more towards the style *with* braces instead of *without* braces.

Reply:
I agree with you, @jeremylt , that the last style is more error-prone and I am happy to stick to the first two if we want to add it to our style convention.

Reply:
I think clang-format can do this automatically if we migrate there at some point.